server/scripts/setupChainEventListeners.ts

```diff
@@ -21,6 +21,13 @@ const log = factory.getLogger(formatFilename(__filename));
 // emit globally any transfer over 1% of total issuance
 // TODO: config this
 const BALANCE_TRANSFER_THRESHOLD_PERMILL: number = 10_000;
+const BALANCE_TRANSFER_THRESHOLD: number = 10 ** 22;
+/* TODO: both of these are imperfect solutions. The BALANCE_TRANSFER_THRESHOLD_PERMILL used
+by the Substrate enricher uses an API call to get the max token amount of the chain its referring to
+for every event it processes. This seems like too much overheard for the ERC20 enricher which
+could end being processed on through hundreds of chains all with rapid transfers happening. So
+I have made it a fixed value here (1000 tokens if they're using the default decimal value) but
+the ideal solution would be to have every chain's max token amount stored in the database. */
```

Comment:
where would you suggest we store these constants? @jnaviask

Reply:
We could fetch the total supply when we convert a token into a chain and then store a threshold that way (even though it may change if tokens are minted/burned, it'll still be roughly in the same order of magnitude). Seems best.